### DISCUSSION TITLE
Recommendation to remove the develop branch

### DISCUSSION THREAD

**Discussion by bsweeney** [dompdf/dompdf#2623] [Project Development]
Back when the develop branch was created I think (?) we were more active. It made sense at that time to have the main branch reflect only the current release to make for easier patching. However, over the years patching hasn't really been a part of the project since we typically just fix urgent issues and release it alongside any other pending changes. At this point maintaining two separate branches does not really provide any benefit to the project. As such I think it's time to drop the develop branch.

Moving forward new development should occur on temporary branches. Once development is complete a pull request should be created to review the code for stability and for merging into master. Any pull requests from outside contributors can be rebased to a new branch if we feel additional work may be necessary before it's ready for the main branch.

**Comment by Mellthas** [comment]
Agree, this would definitely simplify the process. Don’t see any downsides either.

**Reply by bsweeney** [reply]
OK I've merged the current develop branch. I'll start working towards this goal as I reorganize for the next release.
